File: my-folder/0075-sort-colors/solution.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int cs[3];
        memset(cs,0,sizeof(cs));
        
        for(int i=0;i<nums.size();i++)
        {
            cs[nums[i]]++;
        }
        int z=cs[0],o=cs[1],t=cs[2];
        for(int i=0;i<nums.size();i++)
        {
            if(z>0)
            {
                nums[i]=0;
                z--;
            }
            else if(o>0)
            {
                nums[i]=1;
                o--;
            }
            else
            {
                nums[i]=2;
                t--;
            }
        }
        
    }
};

File: my-folder/0075-sort-colors/solution.java
class Solution {
    public void sortColors(int[] nums) {
        int x=0;
        int y=0;
        int z=0;

        for(int num:nums){
            if(num==0)
                x++;
            else if(num==1)
                y++;
            else
                z++;
        }
        System.out.println(x+" "+y);
        for(int i=0;i<nums.length;i++){
            if(x>0){
                nums[i]=0;
                x--;
            }
            else if(y>0){
                nums[i]=1;
                y--;
            }
            else{
                nums[i]=2;
                z--;
            }
        }
    }
}
